### ISSUE TITLE
Can't write numbers in conventional commit scope.

### ISSUE THREAD

**Issue by javimsevilla** [bk201-/jira-prepare-commit-msg#518]
Hi! Thank you for your work creating this tool, it is very useful.

In the [specification of conventional commits](https://www.conventionalcommits.org/) it is not specified whether the scope can contain numbers or not, but if you now have the `isConventionalCommit` property set to `true` and the scope field contains some number in the commit message, then the resulting message is not as expected (as stated in [the README documentation](https://github.com/bk201-/jira-prepare-commit-msg/blob/master/README.md#conventional-commit)).

For example:

With the following configuration:

```
{
  "jira-prepare-commit-msg": {
    "messagePattern": "[$J] $M",
    "isConventionalCommit": true
  }
}
```

If the commit message is:

`test(e2e): add Cypress to project`

The resulting message is:

`[JIRA-4321] test(e2e): add Cypress to project` (BAD)

But the expected result message is:

`test(e2e): [JIRA-4321] add Cypress to project` (EXPECTED)

However, if the commit message is:

`test: add Cypress to project`

or

`test(integration): add Cypress to project`

Then, the result message is:

`test: [JIRA-4321] add Cypress to project` (GOOD)

or

`test(integration): [JIRA-4321] add Cypress to project` (GOOD)

Is this a bug or is it working this way on purpose? Thank you

**Comment by javimsevilla** [comment]
I've created a PR for this 👉 https://github.com/bk201-/jira-prepare-commit-msg/pull/537, please @bk201-  take a look. Thanks.

**Comment by bk201-** [comment]
Hi! I fixed it in 1.7.0. Could you please check your cases?

**Comment by bk201-** [comment]
Now you can use your own conventional pattern
